src/main.cpp

```diff
@@ -623,38 +673,6 @@ int main(int argc, char **argv)
     pidFile << getpid();
     pidFile.close();
 
-	// Load yaml config
```

Comment:
this change is not related to the main topic of the PR. It is in a separate commit. Just to small for a separate PR.